pkg/cloudevents/transport/http/codec_test.go

```diff
@@ -797,13 +797,6 @@ func TestCodecRoundTrip(t *testing.T) {
 					got.Data = data
 				}
 
-				if tc.wantErr != nil || err != nil {
```

Comment:
the tests need to be expanded to non-happy path only. Please leave this?

Reply:
will do, although right now it is unreachable